CHANGELOG.md

```diff
@@ -120,6 +120,10 @@
 
 - PDK now supports getting plugins' ID with `kong.plugin.get_id`.
   [#9903](https://github.com/Kong/kong/pull/9903)
+- **http-log, statsd, opentelemetry, datadog**: The queueing system
+  has been reworked, causing some plugin parameters to be
+  deprecated and new parameters to be added.
```

Comment:
```suggestion
  has been reworked, causing some plugin parameters to not
  function as expected anymore and new parameters to be added.
```

Please move this to the "breaking changes" section (create one if one doesn't exist) and work with @lena-larionova to create a docs page and link to it based on [this](https://github.com/Kong/docs.konghq.com/pull/5088#issuecomment-1489270111) comment.